### PR TITLE
Document watermark-repair subcommand

### LINKED ISSUE #143: Document new subcommand `watermark-repair`
Add a subcommand that will allow adjusting of the slashing protection watermark. This would be useful if all the slashing protection data has been lost and allow web3signer to be safely used to sign again by setting the watermark to a future slot.

See https://github.com/ConsenSys/web3signer/pull/679 and https://github.com/ConsenSys/web3signer/issues/493

### PR DESCRIPTION
**Pull request by alexandratran**
Signed-off-by: Alexandra Tran <alexandratran@protonmail.com>

# Pull request checklist

Use the following template to make sure your PR fits the Web3Signer documentation standard.

## Before creating the PR

Make sure that:

- [x] You've read the [contribution guidelines](https://consensys.net/docs/doctools/).
- [x] You've [previewed your changes locally](https://consensys.net/docs/doctools/en/latest/preview/old-system/#preview-locally).

## Describe the change

Document `watermark-repair` subcommand usage.

## Issue fixed

fixes #143 

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] CircleCI workflow
- [ ] Build and QA tools (for example, lint or vale)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] Read the Docs configuration
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've [fixed any issues](https://consensys.net/docs/doctools/en/latest/contribute/fix-cicd-errors/) raised by the tests.
- [ ] You've [previewed your changes on Read the Docs](https://consensys.net/docs/doctools/en/latest/preview/old-system/#preview-on-read-the-docs)
  and added a [preview link](#preview).

### CODE REVIEW

File: docs/Reference/CLI/CLI-Subcommands.md [hunk 1]
```diff
@@ -1150,6 +1152,96 @@ The file to import the slashing protection database from. The file must be forma
 
 Predefined network configuration. Accepts a predefined network name. The default is `TESTNET`.
 
+### `watermark-repair`
+
+Updates the slashing protection low watermark for validators.
```
Comment (jframe) [inline docs/Reference/CLI/CLI-Subcommands.md:1157]: nit: Maybe we should add a small explanation of what the low watermark is

**Review by jframe — APPROVED** [review]

File: docs/Reference/CLI/CLI-Subcommands.md (hunk 1 above)
Comment (alexandratran) [reply docs/Reference/CLI/CLI-Subcommands.md:1157]: @jframe Can you provide or point me to a good definition of it?

Comment (jframe) [reply docs/Reference/CLI/CLI-Subcommands.md:1157]: Web3Signer will not sign any block that has a slot lower than the block watermark. Similarly, Websigner will not sign any attestations that have a source epoch less than the watermark source epoch or a target epoch less than the watermark target epoch. There are low watermarks for both the block slot and attestation source and targets and these are kept for each validator.

**Review by rolandtyler — APPROVED** [review]
LGTM

**Pull request merged**